perfkitbenchmarker/configs/vpn_tests/005_iperf_vpn_gcp2aws.yml

```diff
@@ -0,0 +1,28 @@
+# VPN iperf config.
```

Comment:
This looks like a specific test, not general functionality?

Reply:
Oops, removed that file.